OpenRA.Mods.Common/Traits/Buildings/BaseProvider.cs

```diff
@@ -38,14 +39,17 @@ public class BaseProvider : ITick, INotifyCreated, IRenderAboveShroudWhenSelecte
 		int total;
 		int progress;
 		bool allyBuildEnabled;
+		bool buildRadiusEnabled;
```

Comment:
Is there a reason why this is not `readonly`?

Reply:
Its not because allyBuildEnabled is not

Reply:
Hm, I think you should make both `readonly` then. ^^
(They only get a value assigned in the constructor.)